mods/pageviews.js

```diff
@@ -320,9 +305,25 @@ PJVS.prototype.pageviewsForProjects = function(restbase, req) {
             }
         }
 
-    });
+    }).catch(notFoundCatcher);
 
-    return dataRequest.then(normalizeResponse);
+    return dataRequest.then(normalizeResponse).then(function(res) {
+        if (res.body.items) {
+            res.body.items.forEach(function(item) {
+                // prefer the v column if it's loaded
+                if (item.hasOwnProperty('v') && item.v !== null) {
```

Comment:
`if (item.v)` should be enough given that by the time this code is deployed, the underlying table will be updated.

Also, it might be a good idea to run a script that goes through all of the records and moves the `views` values into `v`, so that such checks may be removed.